CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # Change Log
 
+## [0.29.0](https://github.com/eclipse/lemminx/milestone/45?closed=1) (November 21, 2024)
+
+### Enhancements
+
+- ICompletionParticipant#onXMLContent is not called for CDATA. See [#1694](https://github.com/eclipse-lemminx/lemminx/issues/1694).
+
+### Bug Fixes
+
+- Supports TextDocumentContentChangeEvent#getRangeLength with null value. See [#1696](https://github.com/eclipse-lemminx/lemminx/pull/1696).
+- org.eclipse.lemminx.dom.DOMNode.getTextContent() seem to be implemented wrong (or incomplete). See [#1695](https://github.com/eclipse-lemminx/lemminx/issues/1695).
```

Comment:
maybe reword to "Implement `org.eclipse.lemminx.dom.DOMNode.getTextContent()` according to the API", similar to how the PR is named

Reply:
renamed